lblod/ember-rdfa-editor-lblod-plugins
# GN-4395: make `errorMessage` of number input modal reactive to attribute changes

### Overview
This PR ensures that the error message shown in the number input popup updates automatically if any node attributes change (such as the minimum and maximum value of the number variable). Currently it is not possible to update the min/max value through a UI when editing the variable, but this PR ensures the error message is updated when changing the html of the document manually.

##### connected issues and PRs:
Solves a large part of https://binnenland.atlassian.net/browse/GN-4395?atlOrigin=eyJpIjoiNjdhZGNkZmRjZGQ3NGY2ZWFkNmY5MzNlMjI5OGMzOWEiLCJwIjoiaiJ9

### How to test/reproduce
- Update the minimum or maximum value of a number variable through the `customHTML` modal and see how a possible error message appears (or disappears).

### Checks PR readiness
